client/header/activity-panel/panels/stock/card.js

```diff
@@ -117,6 +135,17 @@ class ProductStockCard extends Component {
 
 	render() {
 		const { product } = this.props;
+		const { edited, editing } = this.state;
+		const { notifyLowStockAmount } = wcSettings;
+		const lowStockAmount = Number.isFinite( product.low_stock_amount )
+			? product.low_stock_amount
+			: notifyLowStockAmount;
+		const isLowStock = product.stock_quantity < lowStockAmount;
+
+		if ( ! isLowStock && ! edited && ! editing ) {
```

Comment:
It looks like this is the logic to hide the card once the stock has been updated and the panel closed/reopened. Can you add a comment to explain that?

Reply:
Added in 4536f027a1d26120827834e877e482ab1c02b5ee. :+1: